pull_request_template.md

```diff
@@ -1,27 +1,41 @@
 ## Problem
+
 <!-- Why are we making this code change? -->
 
 ## Solution
+
 <!-- How do the changes in this pull request solve the stated problem? Be descriptive. -->
 
 ## Additional Notes
+
 <!-- Is there anything in particular that you want to call attention to? Areas of focus, follow-up actions, etc. -->
 
 ## Links
+
 ### Ticket
+
 <!-- *do not link to private ticketing systems* -->
-GitHub issue _____
+
+GitHub issue:
 
 ### Other links
 
 ## Verification
+
 ### Manual tests
+
 <!-- Include the data and actions taken to exercise the Subject Under Test (SUT). Include any screen captures if relevant. -->
 
 ### Automated tests
+
 - [ ] Unit tests added/updated
 - [ ] E2E tests added/updated
 - [ ] N/A - (provide a reason)
 - [ ] deferred - (provide GitHub issue for tracking)
 
-By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
\ No newline at end of file
+### Housekeeping
+
+- [ ] No non-essential console logs
```

Comment:
This is something we should lint for instead, but I wanted to quickly poll the team before adding to the rule set.